### DISCUSSION TITLE
Diary-10-倒一 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#305] [Announcements]
# Diary-10-倒一 | Zi-Zi's Journey

2024.1.8-2024.01.19

https://gz-metal-cell.github.io/2024/01/13/Diary-10-%E5%80%92%E4%B8%80/

**Comment by promefire** [comment]
# 催更！！！！！！！！

**Reply by GZ-Metal-Cell** [reply]
拍的照片太多了😭努力 P 图中
